.github/workflows/production.yml

```diff
@@ -0,0 +1,100 @@
+name: 🌠 Deploy to production
+
+on: workflow_dispatch
+
+jobs:
+  deploy-server:
+    name: 🚀 Deploy server
+    environment: 
+      name: production
+      url: https://applicants-api.nycplanningdigital.com
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: server
+      - uses: akhileshns/heroku-deploy@v3.13.15
+        name: Deploy server to Heroku
+        with:
+          heroku_email: ${{secrets.HEROKU_EMAIL}}
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: ${{ vars.HEROKU_APP_NAME }}
+          team: ${{secrets.HEROKU_TEAM}}
+          appdir: server
+        env:
+          HD_ADO_PRINCIPAL: ${{ secrets.ADO_PRINCIPAL }}
```

Comment:
What does the `HD` stand for?

Reply:
Heroku Deployment I think? It's from the action we're using

https://github.com/marketplace/actions/deploy-to-heroku#environment-variables